### ISSUE TITLE
Switching Subscriptions Should Prompt User for Confirmation

### ISSUE THREAD

**Issue by BacchusJackson** [indentlabs/notebook#602]
### Expected behavior
As a user, I'm considering switching to the annual plan. I see $7 a month, billed anually and I like the sound of that. I click on it, a window pops up and I see $7 a month is $84 a year, billed once. I click confirm when prompted to change my plan, it display information for my new subscription.

### Actual behavior
I see the subscription options and I'm thinking about switching. I click on another plan and a message pops up saying my plan was successfully changed. Wait! I just wanted to know what $7 a month turns out to be annually! Too late. My card is charged immediately. I switch back and my card is chraged again. Luckily, it gets prorrated for time not used but now my billing history is hard to follow.

tl;dr: Add a confirmation prompt before switching subscriptions with additional plan information.
